client/src/containers/datasets/layers/projects/hooks.tsx

```diff
@@ -34,11 +39,56 @@ export function useLayers({
           '#EFB82A',
           '#176252',
         ],
+        'circle-stroke-opacity': opacity,
         'circle-opacity': opacity,
       },
       layout: {
         visibility: visibility,
       },
+      maxzoom: 6,
     },
-  ] as CircleLayer[];
+    {
+      id: 'projects_boundaries',
+      type: 'line',
+      source: 'projects',
+      'source-layer': 'afoco_locations_full',
+      paint: {
+        // 'line-color': [
+        //   'case',
+        //   ['boolean', ['feature-state', 'hover'], false],
+        //   '#176252',
+        //   '#170099',
+        // ],
+        'line-color': '#176252',
+        'line-opacity': opacity,
+        'line-width': 1.5,
+      },
+      layout: {
+        visibility: visibility,
+      },
+      maxzoom: 18,
+      minzoom: 6,
+    },
+    {
+      id: 'projects_fill',
+      type: 'fill',
+      source: 'projects',
+      'source-layer': 'afoco_locations_full',
+      paint: {
+        // 'fill-color': [
```

Comment:
Do we need this comment code for the future or can we remove?

Reply:
yes!